regulations/generator/node_types.py

```diff
@@ -1,10 +1,12 @@
-#vim: set encoding=utf-8
+# vim: set encoding=utf-8
 from itertools import takewhile
+import re
 
 APPENDIX = u'appendix'
 INTERP = u'interp'
 REGTEXT = u'regtext'
 SUBPART = u'subpart'
+SUBJGRP = u'subjgrp'
```

Comment:
A comment indicating what this is (in terms of a regulation would be useful here). That's probably true of the other things in this list too, but let's start here.